magento/magento2
# Trim email address in customer account create and login form

Related to issue: #6058 

### Description

Trim email address by remove leading or trailing space on following forms.
- Customer account login page (Improved PR code #15365)
- Customer account create page
- Customer authentication popup in case Allow Guest Checkout set to No

### Fixed Issues (if relevant)

1. magento/magento2#6058: IE11 user login email validation fails if field has leading or trailing space

### Manual testing scenarios

1. Open customer account login page in Firefox or IE browser.
2. Try to add space before entering an email address in the Email field.
3. Copy `" johndoe@domain.com "` and paste in the Email field. It will automatically remove leading or trailing space.
4. Check same on customer account create page
5. Check same on customer authentication popup. To check this, set  `Store > Configuration > Sales > Checkout > Checkout Options > Allow Guest Checkout` to `No`. Add product to cart and click on `Proceed to Checkout` button in the cart page or top mini cart.

### Contribution checklist
